### DISCUSSION TITLE
Welcome to wordpress-indieauth Discussions!

### DISCUSSION THREAD

**Discussion by pfefferle** [indieweb/wordpress-indieauth#190] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bydb** [comment]
Hi I want use IAWriter to publish on my  WordPress site which is privatly hosted. Is there a tutorial anywhere? Thanks  for help.

**Reply by dshanske** [reply]
@bydb Assuming IAWriter is using Micropub. Micropub and IndieAuth are pretty much plug and play. The settings are minimal.

**Reply by pfefferle** [reply]
@bydb The usages is really straight forward:

* Install the Micropub Plugin
* Install the IndieAuth Plugin
* Go to the iAWriter Settings -> Accounts
* Hit the `+` Button and choose "Micropub"
* Add the URL of your blog
* Follow the authentication flow
* That's it! 

I hope this works for you.
